src/Twitter.php

```diff
@@ -219,7 +247,7 @@ private function getHttpClient(Configuration $config): HttpClient
      */
     private function getRequestOptions(array $params, string $requestMethod): array
     {
-        unset($params[self::KEY_FORMAT]);
+        unset($params[self::KEY_FORMAT], $params[self::KEY_PREBUILT_ENDPOINT]);
```

Comment:
is `KEY_PREBUILT_ENDPOINT` still required now?

Reply:
Oh no, I forgot about it. Thanks